css-values/Overview.bs

```diff
@@ -2273,7 +2609,35 @@ Changes</h2>
 	Changes since <a href="http://www.w3.org/TR/css-values-3/">Level 3</a>:
 
 	<ul>
-		<li>Added the ''vi'', ''vb'', and ''ic'' units.
```

Comment:
tracking all proposed changes, line by line for maintainability, with Github issue IDs and grouping inside comments